controller/util.py

```diff
@@ -3,12 +3,38 @@
 """
 
 from evdev import ecodes
+import os
+import logging
 
 # dict keys are gotten from /usr/include/linux/input-event-codes.h
 
 DEFAULT_PORT = 5000
 
 
+def init_logger() -> logging.Logger:
```

Comment:
maybe: it'd be nice to have some doc string explaining what this does and/or how to use it like that it should only be used once then reuse said logger, could discuss the specifics